presto-oracle/src/main/java/io/prestosql/plugin/oracle/OracleConfig.java

```diff
@@ -95,4 +98,43 @@ public OracleConfig setTimestampDefaultPrecision(int timestampDefaultPrecision)
         this.timestampDefaultPrecision = timestampDefaultPrecision;
         return this;
     }
+
+    @NotNull
+    public boolean isConnectionPoolEnabled()
+    {
+        return connectionPoolEnabled;
+    }
+
+    @Config("oracle.connection-pool.enabled")
+    public OracleConfig setConnectionPoolEnabled(boolean connectionPoolEnabled)
+    {
+        this.connectionPoolEnabled = connectionPoolEnabled;
+        return this;
+    }
+
+    @Min(1)
```

Comment:
Can this be 0?

Reply:
it shouldnt.. you will not have any connection